3.0/m_roleplay.cpp

```diff
@@ -399,22 +454,22 @@ class CommandBaseRoleplay : public Command
 	 */
 	CmdResult Handle(User* user, const Params& parameters) CXX11_OVERRIDE
 	{
-		Channel* c = ServerInstance->FindChan(parameters[0]);
 		LocalUser* luser = IS_LOCAL(user);
+		if(!luser)
+			// This shouldn't happen.
+			return CMD_FAILURE;
```

Comment:
If the command can only executed by local users consider inheriting from SplitCommand and using HandleUser.

Reply:
Done. :+1: